.github/workflows/validate-schema.yml

```diff
@@ -0,0 +1,65 @@
+name: Schema
+# Validate schemas modified in an pull request or push against their given meta-schema (such as draft-07, etc.)
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - au.org.access-nri/**.json
+  pull_request:
+    branches:
+      - main
+    paths:
+      - au.org.access-nri/**.json
+jobs:
+  setup:
+    name: Setup
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.changed.outputs.all_changed_and_modified_files }}
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Get all modified schema
+      uses: tj-actions/changed-files@48d8f15b2aaa3d255ca5af3eba4870f807ce6b3c  # v45.0.3
+      id: changed
+      with:
+        matrix: true
+        files: |
+          **.json
+
+  validate:
+    name: Validate
+    runs-on: ubuntu-latest
+    needs:
+      - setup
+    strategy:
+      matrix:
+        schema: ${{ fromJson(needs.setup.outputs.matrix) }}
+    env:
+      METASCHEMA_PATH: schema.json
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Download metaschema from schema
+        id: metaschema
+        run: |
+          schema_url=$(jq --compact-output --raw-output '."$schema"' ${{ matrix.schema }})
+
+          if [ -z "$schema_url" ]; then
+            echo '::error::No $schema property present in schema. Please add one to the schema.'
+            exit 1
+          fi
+
+          echo "Metaschema URL: $schema_url"
+          curl -o ${{ env.METASCHEMA_PATH }} $schema_url
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
+
+      - run: jsonschema ${{ env.METASCHEMA_PATH }} -i ${{ matrix.schema }}
```

Comment:
How is `jsonschema` being installed? I ran a local test Python 3.10 environment, and it needed to be installed via `pip`. It might be getting deprecated as well as I got warnings when I was running it  (`DeprecationWarning: The jsonschema CLI is deprecated and will be removed in a future version. Please use check-jsonschema instead, which can be installed from https://pypi.org/project/check-jsonschema/`)

Reply:
`jsonschema` is installed with the above python installation. 
Good to know about it being deprecated...I suppose I might have to `pip install` the new `check-jsonschema`!!

Reply:
We use `jsonschema` a lot...https://github.com/search?q=org%3AACCESS-NRI%20jsonschema&type=code

Reply:
Tested using the new `check-jsonschema` in https://github.com/codegat-test-org/test/actions/runs/11043885031/job/30678883729 and https://github.com/ACCESS-NRI/schema/pull/37/commits/a2062d120d5212c8e9662ffb18120e74efb75945

Reply:
I think it's just the command-line usage that is deprecated? https://pypi.org/project/jsonschema/